### ISSUE TITLE
New term request for mouse branch

### ISSUE THREAD

**Issue by YojanaGadiya** [EBISPOT/efo#1616]
For new term requests, please provide the following information:

## Preferred term label

Humanized mouse

## Synonyms

humanized hCYP1A1_1A2_Cyp1a1/1a2(-/-)_Ahr<d> mutant mice

## Textual definition

The mice carry the human CYP1A1 and CYP1A2 genes in the absence of functional mouse Cyp1a1 and Cyp1a2 orthologs and mimic the human poor-affinity aryl hydrocarbon receptor (AHR) by carrying the poor-affinity Ahrd allele derived from DBA/2J mice. These may be useful in drug or carcinogen metabolism research; specifically as a model for human risk assessment studies involving drug or environmental toxicants that may be substrates for the aryl hydrocarbon receptor (AHR) or cytochrome P450 family 1 members. A mouse strain is described in Jackson Laboratory https://www.jax.org/strain/008599 

## Suggested parent term

Mus musculus http://purl.obolibrary.org/obo/NCBITaxon_10090 

## Attribution

0000-0002-7683-0452

**Comment by zoependlington** [comment]
Thanks @YojanaGadiya! I'll get this into EFO before the June (15th) release so it will be available after then. I'll reply to this ticket with the ID when it is minted.

**Comment by zoependlington** [comment]
Term has been added as EFO_0021426.
